docs/Reference/Reply.md

```diff
@@ -7,7 +7,6 @@
   - [.statusCode](#statuscode)
   - [.server](#server)
   - [.header(key, value)](#headerkey-value)
-      - [set-cookie](#set-cookie)
```

Comment:
I don't get it why of this change

Reply:
It is weird, VSCode was constantly deleting the line on save even that I don't have the fix-on-save enabled 🤔

Reply:
```jsonc
{
  "editor.formatOnSave": false,
  // here is the trick to prevent any action
  "editor.codeActionsOnSave": {}
}
```

Reply:
Thanks! Didn't expect to VSCode do that without telling it 😅